src/test/java/com/karankumar/bookproject/backend/service/UserServiceTest.java

```diff
@@ -31,36 +30,27 @@
 
 import javax.validation.ConstraintViolationException;
 
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static com.karankumar.bookproject.utils.SecurityTestUtils.*;
```

Comment:
As per our style guide, we are not using wildcard imports. Could you please use explicit imports, instead?